manifests/init.pp

```diff
@@ -155,6 +155,12 @@
 # @param api_client_auth_cn_map
 #   Mapping of certificate common name and Pulp user to authenticate to Pulp API.
 #
+# @param pulp_cache_enable
```

Comment:
If the goal is to have an exact correspondence with the name of the settings in settings.py (which is a fine approach) then I suppose it should be `pulp_cache_enabled` ?

Personally I am a bit more biased towards names that are concise and simple to the user, and would suggest `enable_content_caching` (the "pulp" part is clear from context) or simply `content_caching`

Reply:
I can go with `pulp_cache_enabled` to match the setting. I think discoverability wise it would be easier for those that look at Pulp for these to match. Now, over on the puppet-foreman_proxy_content side where we wrap these options, that can be a bit more nuanced.

Reply:
I left a note at https://github.com/pulp/pulpcore/pull/1356/files#r661030668 that the `pulp_` prefix is weird to me. I see @ehelms' point about matching names so let's make sure upstream picks a good one.